contracts/Market.sol

```diff
@@ -347,29 +349,17 @@ contract Market is PausableAccessPreset {
   }
 
   /**
-   * @dev Reverts if supplier is out of stock or if total available supply in the market is being reserved for priority buyers
-   * and buyer is not priority.
+   * @dev Reverts if supplier is out of stock or if total available supply in the market is being reserved for priority
+   * buyers and buyer is not priority.
    *
-   * @param certificateAmount The number of carbon removals being purchased.
+   * @param supplierAddress The supplier address to check.
    */
-  function _checkSupplyOfSupplier(
-    uint256 certificateAmount,
-    address supplierAddress
-  ) private view {
+  function _checkSupplyOfSupplier(address supplierAddress) private view {
     uint256 activeSupplyOfSupplier = _activeSupply[supplierAddress]
       .getTotalBalanceFromRemovalQueue(_removal);
     if (activeSupplyOfSupplier == 0) {
       revert OutOfStock();
     }
-    if (certificateAmount > activeSupplyOfSupplier) {
-      revert InsufficientSupply(); // todo Assure `_checkSupplyOfSupplier` validates all possible market supply states
```

Comment:
I also don't see why we want to get rid of this

Reply:
@amiecorso  is this stale now that I've merged in your changes?

Reply:
my other changes didn't address this... but the new diff I just introduced does https://github.com/nori-dot-eco/contracts/pull/329